README.md

```diff
@@ -1,4 +1,108 @@
 # OpenFeature project governance
 
-> :warning: Details are coming soon!
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/openfeatureflags/projects/1)
+[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/openfeatureflags/.github/blob/main/CONTRIBUTING.md)
+
+This document describes the bootstrap governance process under which the project will operate
+until the final governance process is identified.
+
+> :warning: This is a temporary (aka bootstrap) governance document that
+> is effective until the project is fully established.
+> A final process 
+
+## Project manifesto
+
+### Our goal
+
+Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
+
+### Scope
+
+The scope includes but not limited to:
+
+- An **open standard** that allows adopters and vendors to easily integrate with spec-compliant feature flag solutions
+- Creating a **vendor-agnostic ecosystem** supported by the CNCF that provide a seamless experience for developers, SREs, and vendors
+- Providing Unified API and SDK;
+- Developer-first, cloud-native reference implementation; 
+- Ensuring extensibility for open source and commercial offerings.
+
+### Why OpenFeature?
+
+There are many existing solutions, open source or not.
+Unfortunately they lack key features needed for wide adoption
+by companies and in the open source ecosystem:
+
+- **Many SDKs** doing the same thing, but slightly differently
+- Many companies **“rebuilding” the wheel**
+- SDKs are generally open source but have no ecosystem
+- **“Vendor” lock-in** at code-level
+- **No collaboration** across vendors. 
+- **Missing integration** with observability tools (e.g. OpenTelemetry)
+
+### Project Differentiators
+
+This project is more than just copy and paste of existing solutions
```

Comment:
These bullet points are all negatives of existing solutions. I would propose to rephrase them to make them positives of your proposed solution. For example instead of saying existing solutions have eval logic in SDKs, you could say that openfeature will evaluate all rules on a centralized service. Or instead of saying existing solutions do not have a unified grammar, you could say that you are proposing a unified grammar specification that may be understood by any spec-compliant tooling.

Reply:
This comes from the slidedeck by @beeme1mr. I do not have strong opinion here, because manifesto documents are usually quite opinionated. Happy to rephrase though